android/src/main/java/com/thebluealliance/androidclient/helpers/ThreadSafeFormatters.java

```diff
@@ -55,4 +55,8 @@ public static synchronized String formatDoubleOnePlace(double input) {
     public static synchronized String formatDoubleTwoPlaces(double input) {
         return DOUBLE_TWO_PLACES_FORMAT.format(input);
     }
+
+    public static synchronized String formatStat(double stat) {
```

Comment:
It's an easy enough change to make in the future should we need it, I'd rather just keep simplicity in the code now. Plus, at the call side, `formatDoubleTwoPlaces` is more descriptive about what's happening than `formatStat` is